components/About/BibleVerse.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { Container } from "react-bootstrap"
+
+export default function BibleVerse(props:any) {
```

Comment:
I think it was a good callout from Jay in the other PR? that we should start defining the types, rather than using 'any'

Reply:
Ooo good catch, that's a quick fix!